project/mmi366127/README.md

```diff
@@ -0,0 +1,107 @@
+# Geometric search engine
+
+A project that implement R-tree, kd-tree, M-tree for geometric searching.
+
+## Basic Information
+GitHub repository: [Geometric search engine](https://github.com/mmi366127/Geometric-Search-Engine)
+
+Geometric search is important for implmenting map and geometry system. For more efficient query, there are many data structures for different circumstances, including R-tree, kd-tree and M-tree. Moreover, these data structures can also be used to do machine learning tasks in KNN(K-nearest-neighbors) algorithm. Therefore, in this project, I am going to implement an API for geometric search for arbitrary dimension data. 
+
+## Problem to Solve
+
+Using R-tree, kd-tree and M-tree to do query on nearest-n-points, and range-query. This project assumes using Euclidean distance for distance metric(could add more distance metric if possible.)
```

Comment:
Do you want to use 3 trees at once?  Please make it clear.

Reply:
I think I would only provide them separately.

Reply:
That implies you will make 3 implementations separately.  It would be interesting but I'd like to remind that it's also quite time-consuming.  I hope we have time to discuss about it face to face.